include/capstone/capstone.h

```diff
@@ -84,6 +84,8 @@ typedef enum cs_arch {
 	CS_ARCH_TMS320C64X,	///< TMS320C64x architecture
 	CS_ARCH_M680X,		///< 680X architecture
 	CS_ARCH_EVM,		///< Ethereum architecture
+	CS_ARCH_RISCV,          ///< RISCV architecture
```

Comment:
so this introduces 2 new architecture enums? can we combine them into 1, and allow selection with CS_MODE_xxx?

Reply:
CS_ARCH_RISCVC has a different alignment and instruction coding size even opcode.
If we combine how to deal with `uint8_t skipdata_size(cs_struct *handle)` ?

Reply:
cs_struct has all the arch & mode info, so you can check the mode, and act accordingly?

Reply:
we did similar thing with ARM: ARM & Thumb are both under the same arch CS_ARCH_ARM, but different mode CS_MODE_ARM & CS_MODE_THUMB. i suppose this can do the same thing?

Reply:
Aha, you are right, I saw the ARM/thumb code.